src/main/resources/avro/MAPRecord.avsc

```diff
@@ -1132,6 +1132,21 @@
             "type": "array",
             "items": "string"
         }
+    },
+    {
+      "name": "iiifManifest",
+      "type": [
+        "null",
+        "string"
+      ]
+    },
+    {
+      "name": "mediaMaster",
+      "type": {
+        "name": "mediaMasterArray",
+        "type": "array",
+        "items": "string"
+      }
```

Comment:
I would have thought this already got updated when you started implementing the mappings. did this creep in from somewhere else?

Reply:
I missed it in the mapping -- this is what took me forever to debug yesterday. Why was it getting mapped but not showing up in the JSONL.